src/systemctl/systemctl.c

```diff
@@ -7804,6 +7805,10 @@ static int systemctl_help(void) {
                "     --boot-loader-entry=NAME\n"
                "                         Boot into a specific boot loader entry on next boot\n"
                "     --plain             Print unit dependencies as a list instead of a tree\n"
+               "     --timestamp=FORMAT  Change format of printed timestamps.\n"
+               "                         'pretty' (default): 'Day YYYY-MM-DD HH:MM:SS TZ\n"
+               "                         'us', 'µs': 'Day YYYY-MM-DD HH:MM:SS.UUUUUU TZ\n"
```

Comment:
i'd probably not show µs here. I mean, I love unicode, but in the brief version for --help we don't need to mention this, i.e. redundancy should be removed in --help text...

(it makes sense to mention this in the man page, i.e. the long form)

Reply:
removed